docs/how-to-use-minio-as-laravel-file-storage.md

```diff
@@ -4,7 +4,7 @@
 
 ## 1. Prerequisites
 
-Install MinIO Server from [here](https://www.minio.io/downloads.html).
+Install MinIO Server from [here](https://www.min.io/downloads.html).
```

Comment:
We can give the right link to avoid redirects.

Reply:
whats the right link
```suggestion
Install MinIO Server from [here](https://www.min.io/download.html).
```

Reply:
is this what you mean?

Reply:
https://min.io/download - just this

Reply:
```suggestion
Install MinIO Server from [here](https://www.min.io/download).
```